addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -359,6 +359,8 @@ export class WysiwygAdapterComponent extends Wysiwyg {
                 ...options.toolbarOptions,
                 showChecklist: false,
                 showAnimateText: true,
+                showFontSize: false,
```

Comment:
I'm surprised to see this here, especially with a value of `false`. I'm not saying it's wrong, I'm merely surprised. Could you explain, please ?

Reply:
Not setting it false here would result in `showFontSize` being true always true which inturn will make input box readonly for website which we do not want.

Reply:
But the input being readonly or not should be about the _custom_ font size, right ? The input is specifically for users to write their own custom font size.

Reply:
@dmo-odoo yes and we agreed to put custom font size option true only for website.And since toolbar which website adapts original toolbar which has showFontSize set to true, i made two changes:
1) set showFontSize to false for website just to avail customFontSize to be true.
2) made inputbox readonly depending which (showFontSize or customFontSize) is true.
should there be any other changes?